### PR TITLE
fixup sidebar, modidy parts in plugins/presets

### PR DESCRIPTION
**Pull request by hzoo**
changed sidebar a bit, will do more in a next pr.
- showing the full name of package (removed the sidebar_label, which caused some misc formatting but should be ok)

image

- only updated docs/presets and doc/plugins pages so can ignore the others mostly.
- Fixes https://github.com/babel/website/issues/2179 (just change javascript to typescript code block in 2 places)

### CODE REVIEW

File: _redirects [hunk 1]
```diff
@@ -15,6 +15,9 @@ https://babel.netlify.com/* https://babeljs.io/:splat 301!
 /7/                                 https://old.babeljs.io/repl/build/7.0/
 /7.html                             https://old.babeljs.io/repl/build/7.0/
 
+/docs/en/plugins#pluginpreset-paths /docs/en/plugins#using-a-plugin
+/docs/en/presets#preset-paths /docs/en/presets#using-a-preset
```
Comment (hzoo) [inline _redirects:19]: not sure if this works?

File: docs/plugins.md [hunk 2]
```diff
@@ -97,37 +113,6 @@ These plugins apply transformations to your code.
 - [throw-expressions](plugin-proposal-throw-expressions.md)
 - [private-property-in-object](plugin-proposal-private-property-in-object.md)
 
-### Minification
```
Comment (hzoo) [inline docs/plugins.md:100]: we probably don't need to recommend this

File: docs/plugins.md [hunk 3]
```diff
@@ -169,48 +154,6 @@ Your `.babelrc`:
...
-
-You can also specify an relative/absolute path to your plugin.
-
-```json
-{
-  "plugins": ["./node_modules/asdf/plugin"]
-}
-```
-
-### Plugin Shorthand
```
Comment (hzoo) [inline docs/plugins.md:190]: this is confusing and we recommend using the full path now anyway so can remove? https://github.com/babel/website/issues/2244

Have a link to options.md#name-normalization (it's sorta hidden) but it's there

File: _redirects (hunk 1 above)
Comment (nicolo-ribaudo) [reply _redirects:19]: I don't think so, the hash is not sent to the server.
We can add an empty div with `id="preset-paths"` next to the `using-a-preset` section.

**Review by nicolo-ribaudo — COMMENTED** [review]
Thanks!

**Review by nicolo-ribaudo — APPROVED** [review]

File: _redirects (hunk 1 above)
Comment (hzoo) [reply _redirects:19]: Awesome! 
It's a little below the section, but it's not a big deal if we don't either way I guess?

**Pull request merged**